bin/git-reauthor

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bash
+
+
+init_variables() {
+  COMMAND=${0#*-}
+
+  CONFIG=false
+  ALL=false
+  unset OLD_EMAIL
+  unset CORRECT_EMAIL
+  unset CORRECT_NAME
+  TYPE='full'
+}
+
+
+usage() {
+  cat << EOF
+usage: git ${COMMAND} [<options>]
+
+Options
+    -A, --all                     rewrite all commits and tags
+    -C, --config                  define correct values from user Git config
```

Comment:
changed to `--use-config`, in commit 614d6ab (no need to specify git I think)
removed capital letters in short flags, in commit a6058f3 (was a convention of mine to distinguish flags)